### DISCUSSION TITLE
Question 77: What is the best way to indicate that a group of specimens went together? [JD, SK - IZ]

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#1155] [Q&A] [answered]
Description: We have a group of specimens and documents that appear to be from a single accession, but the available information is insufficient to determine with certainty which accession they came from, or whether they pertain to an accession not recorded in EMu. We would like to record the association of these items with one another in some way, so that the information is not lost, in the hopes that someone will eventually be able to reunite the specimens with fuller data. Is this best recorded simply by explaining the situation with a note, or is there a better way?

**Comment by rondlg** [comment, marked as answer]
Response: Use an event record to record the information that leads you to believe the specimens go together and associate their catalogue records in a group on the Event 2 tab in the Associated Records [EveAssociatedRecords_tab] field. 

Chris M. do we have the procedure documented from the PaleoVerts transaction records? [CM] To share the documentation when we have it.

[Events module information and workflows](https://github.com/fieldmuseum/EMu-Documentation/wiki/eevents-module)

**Comment by cmaves3** [comment]
Here is the link to the Documentation:
https://docs.google.com/document/d/1C1O-rxEIYERgoZxJocmZLMMvgak5Eznsn4mNKB7sAnE
